System.Runtime.Serialization.Formatters CI failure.

## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=795917
Build error leg or test failing: System.Runtime.Tests.WorkItemExecution
Pull request: https://github.com/dotnet/runtime/pull/107268
<!-- Error message template  -->
## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssueJsonStepByStep.md).

<!-- Use ErrorMessage for String.Contains matches. Use ErrorPattern for regex matches (single line/no backtracking). Set BuildRetry to `true` to retry builds with this error. Set ExcludeConsoleLog to `true` to skip helix logs analysis. -->

```json
{
  "ErrorMessage": "publish/ProxyProjectForAOTOnHelix.proj(116,5): error MSB3030: Could not copy the file",
  "ErrorPattern": "error MSB3030: Could not copy the file \"\/tmp\/helix\/working\/[^\\\/]+\/p\/build\/microsoft\\.netcore\\.app\\.runtime\\.[^\\\/]+-arm64\/runtimes\/[^\\\/]+-arm64\/native\/System\\.Runtime\\.Serialization\\.Formatters\\.pdb\" because it was not found.",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```


<!-- Known issue validation start -->
 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=795917
**Error message validated:** `[error MSB3030: Could not copy the file "/tmp/helix/working/[^\/]+/p/build/microsoft\.netcore\.app\.runtime\.[^\/]+-arm64/runtimes/[^\/]+-arm64/native/System\.Runtime\.Serialization\.Formatters\.pdb" because it was not found.`]
**Result validation:** :white_check_mark: Known issue matched with the provided build.
**Validation performed at:** 9/3/2024 7:12:12 PM UTC
<!-- Known issue validation end -->
<!--Known issue error report start -->

### Report

|Build|Definition|Test|Pull Request|
|---|---|---|---|
|[798347](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798347)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798347&view=ms.vss-test-web.build-test-results-tab&runId=20573752&resultId=100007)|dotnet/runtime#105427|
|[797804](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797804)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797804&view=ms.vss-test-web.build-test-results-tab&runId=20572074&resultId=100007)|dotnet/runtime#107368|
|[798274](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798274)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798274&view=ms.vss-test-web.build-test-results-tab&runId=20571782&resultId=100007)|dotnet/runtime#107385|
|[798212](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798212)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798212&view=ms.vss-test-web.build-test-results-tab&runId=20568942&resultId=100001)|dotnet/runtime#107278|
|[798182](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798182)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798182&view=ms.vss-test-web.build-test-results-tab&runId=20567510&resultId=100007)|dotnet/runtime#105771|
|[797909](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797909)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797909&view=ms.vss-test-web.build-test-results-tab&runId=20567184&resultId=100007)|dotnet/runtime#107009|
|[797898](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797898)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797898&view=ms.vss-test-web.build-test-results-tab&runId=20566664&resultId=100001)|dotnet/runtime#107372|
|[798102](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798102)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798102&view=ms.vss-test-web.build-test-results-tab&runId=20563580&resultId=100001)|dotnet/runtime#107378|
|[798015](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798015)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=798015&view=ms.vss-test-web.build-test-results-tab&runId=20560008&resultId=100001)|dotnet/runtime#107376|
|[797972](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797972)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797972&view=ms.vss-test-web.build-test-results-tab&runId=20558810&resultId=100001)|dotnet/runtime#106843|
|[797961](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797961)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797961&view=ms.vss-test-web.build-test-results-tab&runId=20558384&resultId=100001)|dotnet/runtime#107369|
|[797906](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797906)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797906&view=ms.vss-test-web.build-test-results-tab&runId=20556510&resultId=100007)|dotnet/runtime#106873|
|[797572](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797572)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797572&view=ms.vss-test-web.build-test-results-tab&runId=20555452&resultId=100001)|dotnet/runtime#107357|
|[797734](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797734)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797734&view=ms.vss-test-web.build-test-results-tab&runId=20555384&resultId=100007)|dotnet/runtime#107366|
|[797859](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797859)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797859&view=ms.vss-test-web.build-test-results-tab&runId=20555184&resultId=100007)|dotnet/runtime#106781|
|[797850](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797850)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797850&view=ms.vss-test-web.build-test-results-tab&runId=20554946&resultId=100007)||
|[797789](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797789)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797789&view=ms.vss-test-web.build-test-results-tab&runId=20553334&resultId=100001)|dotnet/runtime#107231|
|[796892](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796892)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796892&view=ms.vss-test-web.build-test-results-tab&runId=20552866&resultId=100001)|dotnet/runtime#107267|
|[797654](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797654)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797654&view=ms.vss-test-web.build-test-results-tab&runId=20550490&resultId=100007)|dotnet/runtime#107360|
|[797591](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797591)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797591&view=ms.vss-test-web.build-test-results-tab&runId=20548852&resultId=100007)|dotnet/runtime#107333|
|[797585](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797585)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797585&view=ms.vss-test-web.build-test-results-tab&runId=20548738&resultId=100007)|dotnet/runtime#107358|
|[797496](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797496)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797496&view=ms.vss-test-web.build-test-results-tab&runId=20546036&resultId=100007)|dotnet/runtime#107351|
|[797455](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797455)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797455&view=ms.vss-test-web.build-test-results-tab&runId=20545416&resultId=100007)|dotnet/runtime#107349|
|[797341](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797341)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797341&view=ms.vss-test-web.build-test-results-tab&runId=20543966&resultId=100007)|dotnet/runtime#106875|
|[797272](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797272)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797272&view=ms.vss-test-web.build-test-results-tab&runId=20538716&resultId=100007)|dotnet/runtime#107185|
|[797304](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797304)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797304&view=ms.vss-test-web.build-test-results-tab&runId=20539930&resultId=100007)|dotnet/runtime#107291|
|[797279](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797279)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797279&view=ms.vss-test-web.build-test-results-tab&runId=20539860&resultId=100007)|dotnet/runtime#106977|
|[797152](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797152)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797152&view=ms.vss-test-web.build-test-results-tab&runId=20538236&resultId=100007)|dotnet/runtime#107325|
|[797203](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797203)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797203&view=ms.vss-test-web.build-test-results-tab&runId=20536744&resultId=100007)|dotnet/runtime#107193|
|[796060](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796060)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796060&view=ms.vss-test-web.build-test-results-tab&runId=20534058&resultId=100007)|dotnet/runtime#105427|
|[797168](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797168)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797168&view=ms.vss-test-web.build-test-results-tab&runId=20533982&resultId=100007)||
|[797145](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797145)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797145&view=ms.vss-test-web.build-test-results-tab&runId=20533548&resultId=100007)|dotnet/runtime#107196|
|[797063](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797063)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797063&view=ms.vss-test-web.build-test-results-tab&runId=20530118&resultId=100007)|dotnet/runtime#107334|
|[797040](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797040)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=797040&view=ms.vss-test-web.build-test-results-tab&runId=20529612&resultId=100007)|dotnet/runtime#107333|
|[796999](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796999)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796999&view=ms.vss-test-web.build-test-results-tab&runId=20528344&resultId=100007)|dotnet/runtime#107176|
|[796782](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796782)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796782&view=ms.vss-test-web.build-test-results-tab&runId=20528240&resultId=100007)|dotnet/runtime#107318|
|[796888](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796888)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796888&view=ms.vss-test-web.build-test-results-tab&runId=20525874&resultId=100001)|dotnet/runtime#89204|
|[796871](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796871)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796871&view=ms.vss-test-web.build-test-results-tab&runId=20525114&resultId=100001)|dotnet/runtime#107166|
|[796831](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796831)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796831&view=ms.vss-test-web.build-test-results-tab&runId=20523844&resultId=100007)|dotnet/runtime#107219|
|[795886](https://dev.azure.com/dnceng-public/public/_build/results?buildId=795886)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=795886&view=ms.vss-test-web.build-test-results-tab&runId=20522116&resultId=100001)|dotnet/runtime#107266|
|[796768](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796768)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796768&view=ms.vss-test-web.build-test-results-tab&runId=20522264&resultId=100001)|dotnet/runtime#107316|
|[796425](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796425)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796425&view=ms.vss-test-web.build-test-results-tab&runId=20521324&resultId=100007)|dotnet/runtime#106771|
|[796441](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796441)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796441&view=ms.vss-test-web.build-test-results-tab&runId=20520526&resultId=100007)|dotnet/runtime#107301|
|[796715](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796715)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796715&view=ms.vss-test-web.build-test-results-tab&runId=20521020&resultId=100007)|dotnet/runtime#107310|
|[796691](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796691)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796691&view=ms.vss-test-web.build-test-results-tab&runId=20519978&resultId=100007)||
|[796653](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796653)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796653&view=ms.vss-test-web.build-test-results-tab&runId=20519030&resultId=100007)|dotnet/runtime#107196|
|[796647](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796647)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796647&view=ms.vss-test-web.build-test-results-tab&runId=20518802&resultId=100007)|dotnet/runtime#107307|
|[796613](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796613)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796613&view=ms.vss-test-web.build-test-results-tab&runId=20518148&resultId=100007)|dotnet/runtime#106977|
|[796617](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796617)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796617&view=ms.vss-test-web.build-test-results-tab&runId=20517840&resultId=100007)|dotnet/runtime#106658|
|[796474](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796474)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796474&view=ms.vss-test-web.build-test-results-tab&runId=20514696&resultId=100007)|dotnet/runtime#104637|
|[795917](https://dev.azure.com/dnceng-public/public/_build/results?buildId=795917)|dotnet/runtime|[System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=795917&view=ms.vss-test-web.build-test-results-tab&runId=20511506&resultId=100007)|dotnet/runtime#107268|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|29|51|51|
<!--Known issue error report end -->

--- a/src/mono/msbuild/apple/data/ProxyProjectForAOTOnHelix.proj
+++ b/src/mono/msbuild/apple/data/ProxyProjectForAOTOnHelix.proj
@@ -109,8 +109,8 @@
           Condition="'$(UsePortableRuntimePack)' == 'true'">
 
     <ItemGroup>
-      <_RuntimePackFiles Condition="%(_AppleUsedRuntimePackFiles.Extension) == '.dll' and %(_AppleUsedRuntimePackFiles.FileName) != 'System.Private.CoreLib'" Include="@(_AppleUsedRuntimePackFiles->'$(MicrosoftNetCoreAppRuntimePackLibDir)%(FileName)%(Extension)')" />
-      <_RuntimePackFiles Condition="%(_AppleUsedRuntimePackFiles.Extension) != '.dll' or %(_AppleUsedRuntimePackFiles.FileName) == 'System.Private.CoreLib'" Include="@(_AppleUsedRuntimePackFiles->'$(MicrosoftNetCoreAppRuntimePackNativeDir)%(FileName)%(Extension)')" />
+      <_RuntimePackFiles Condition="(%(_AppleUsedRuntimePackFiles.Extension) == '.dll' or %(_AppleUsedRuntimePackFiles.Extension) == '.pdb') and %(_AppleUsedRuntimePackFiles.FileName) != 'System.Private.CoreLib'" Include="@(_AppleUsedRuntimePackFiles->'$(MicrosoftNetCoreAppRuntimePackLibDir)%(FileName)%(Extension)')" />
+      <_RuntimePackFiles Condition="(%(_AppleUsedRuntimePackFiles.Extension) != '.dll' and %(_AppleUsedRuntimePackFiles.Extension) != '.pdb') or %(_AppleUsedRuntimePackFiles.FileName) == 'System.Private.CoreLib'" Include="@(_AppleUsedRuntimePackFiles->'$(MicrosoftNetCoreAppRuntimePackNativeDir)%(FileName)%(Extension)')" />
     </ItemGroup>
 
     <Copy SourceFiles="@(_RuntimePackFiles)"

